patent.py

```diff
@@ -0,0 +1,40 @@
+import argparse
+import requests
+import json
+
+
+def patent(**kwargs):
+    patent_url = 'https://api.nasa.gov/patents/content'
+    get_params = {}
+    for key, value in kwargs.iteritems():
+        get_params[key] = value
+    get_response = requests.get(patent_url, params = kwargs)
+
+    if get_response.status_code == 429:
+        raise Exception('You have exceeded your rate limit')
+
+    body = get_response.json()
+
+    if 'error' in body:
+        raise Exception(body['error'])
+# Current default is to print out json, need to add ability for file output
+    pretty_print = json.dumps(body, indent=2)
+    return pretty_print
+
+
+def main():
```

Comment:
This is a good idea, but not the right place for this. This file won't be run directly by the user, but by other Python modules, so anything related to arguments should go elsewhere. I'd remove this function, and everything below it, as this module needs to focus exclusively on functions for the Patents API.